### DISCUSSION TITLE
How can I disable throttling check?

### DISCUSSION THREAD

**Discussion by realtebo** [Laravel-Backpack/community-forum#624] [General] [answered]
I am trying the password reset feature and I need to be able to try it a lot of time / minutes without any checks.

How can i disable this check [in local only]

**Comment by promatik** [comment, marked as answer]
Hey @realtebo!

To temporarily disable throttle locally you can comment the throttle line on `App\Http\Kernel.php`, or you can do it for specific routes;

```php
Route::post('/login', [AuthController::class, 'login'])->withoutMiddleware('throttle:120,1');
```
